tests/e2e-test/internal/runner/config.go

```diff
@@ -0,0 +1,49 @@
+package runner
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"gopkg.in/yaml.v3"
+)
+
+// TestConfig contains the information needed to run a test. Each test case
+// run by this driver is described by a `TestConfig`.
+//
+// Example of a test config:
+//
+// 	pkgPath: foo/bar
+//	network: true
+type TestConfig struct {
+	// PkgPath is the path to the package which will be tested.
+	PkgPath string `json:"pkgPath,omitempty" yaml:"pkgPath,omitempty"`
+	// Network indicates whether enable network access for function or not
+	Network bool `json:"network,omitempty" yaml:"network,omitempty"`
```

Comment:
This is the only test data that's not contained within the package itself. Why not move this into the package so all it takes to test a package is the path?

You can then get rid of TestConfig all together.